radix-engine/src/vm/wasm/wasmi.rs

```diff
@@ -1305,3 +1309,49 @@ impl WasmEngine for WasmiEngine {
         instance
     }
 }
+
+#[cfg(feature = "radix_engine_tests")]
+pub fn run_module_with_mutable_global(
```

Comment:
I know that this function makes no sense without `test_wasm_non_mvp_mutable_globals_execute_code()` from `radix-engine-tests/tests/wasm_non_mvp.rs`
But decided to put it here since it is easier to find some references when using `wasmi` module.

Reply:
Moved the whole `test_wasm_non_mvp_mutable_globals_execute_code()` test to the `wasmi.rs`